modules/meeting/app/controllers/meetings_controller.rb

```diff
@@ -97,8 +96,8 @@ def create # rubocop:disable Metrics/AbcSize
 
     if call.success?
       text = I18n.t(:notice_successful_create)
-      if User.current.time_zone.nil?
-        link = I18n.t(:notice_timezone_missing, zone: Time.zone)
+      unless User.current.pref.time_zone?
+        link = I18n.t(:notice_timezone_missing, zone: formatted_time_zone_offset)
         text += " #{view_context.link_to(link, { controller: '/my', action: :settings, anchor: 'pref_time_zone' },
                                          class: 'link_to_profile')}"
```

Comment:
I tried it. When there is no time zone set for the user, the message is displayed inviting the user to set a time zone, but the notification disappears after 5 seconds (or so). **This is extremely short to read and take action**. I suggest the success toast should not autoclose in this case. Probably worth a new bug work package for it.

Reply:
I created a [ticket for this](https://community.openproject.org/wp/58002). There are quite a number of places where we present times and not always do we tell the user that it might not be in their local time (e.g. work package activities). So I think there should be some nudging whenever the time zone is not set.